store/models/common.go

```diff
@@ -184,13 +184,10 @@ func (j JSON) Merge(j2 JSON) (JSON, error) {
 		cleaned[k] = v.Value()
 	}
 
-	b, err := json.Marshal(cleaned)
-	if err != nil {
-		return JSON{}, err
-	}
-
+	b, _ := json.Marshal(cleaned)
```

Comment:
Seems problematic to drop this error. Something I'm missing?

Reply:
This code cannot produce an error.